loadtest/control/simulcontroller/controller.go

```diff
@@ -58,6 +58,7 @@ func (c *SimulController) Run() {
 			c.status <- c.newErrorStatus(err)
 		}
 		c.user.Cleanup()
+		c.user.ClearUserData()
```

Comment:
That's an interesting one. Is this actually needed or does it simply help the GC to re-collect the memory more quickly?

Reply:
This is needed in cases when the coordinator stops an agent, and then starts it again. So the agent never shuts down, and unless the store is cleared, will leak memory.

Reply:
Oh you mean the entire controller struct goes out of scope, is it ? Yeah then I guess it's not needed. But maybe good to add it anyways.

Reply:
Yeah, fine with keeping it. Just thought that when we are done with the controller we remove it from the slice and the GC should do the rest.